src/SimplCommerce.Infrastructure/Data/IRepositoryWithTypedId.cs

```diff
@@ -11,6 +12,9 @@ public interface IRepositoryWithTypedId<T, TId> where T : IEntityWithTypedId<TId
 
         void Add(T entity);
 
+        Task AddRangeAsync(IEnumerable<T> entity);
```

Comment:
Remove 1 blank line (current is 2)
We should use the AddRange instead. Below is the description of AddRangeAsync

       //     This method is async only to allow special value generators, such as the one
        //     used by 'Microsoft.EntityFrameworkCore.Metadata.SqlServerValueGenerationStrategy.SequenceHiLo',
        //     to access the database asynchronously. For all other cases the non async method
        //     should be used.